src/components/Block/Space.vue

```diff
@@ -1,9 +1,10 @@
 <script setup>
-import { computed } from 'vue';
+import { computed, ref, watchEffect } from 'vue';
 import { getInstance } from '@snapshot-labs/lock/plugins/vue3';
 import { useWeb3 } from '@/composables/useWeb3';
 import { useApp } from '@/composables/useApp';
-
+import { useSpaceSubscription } from '@/composables/useSpaceSubscription';
+import { useFollowSpace } from '@/composables/useFollowSpace';
```

Comment:
We should separate import and code with an empty line
```suggestion
import { useFollowSpace } from '@/composables/useFollowSpace';

```